docs/source/use-cases.rst

```diff
@@ -395,12 +395,22 @@ Streaming data engineering
 
 A data engineer responsible for watching a data feed needs to scale out a
 continuous process.  They `combine dask.distributed with normal Python Queues`_ to
-produce a rudimentary but effective stream processing system.
+produce a rudimentary but effective stream processing system. Alternatively, futures can be 
+streamed beween different clients using `channels`_ .
 
 Because dask.distributed is elastic, they can scale up or scale down their
 cluster resources in response to demand.
 
+End user application development
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dask and Distributed offer a 100% Python alternative to message queue based tools such as `Celery`_. 
+The developer who is looking to support power users for custom plugins but also basic users who need
+to run long running, pre-baked tasks can use distributed `channels`_ . This allows the distributed Client to 
+live only for the duration of a web request and the task status to be retrieved later or sent over a websocket.
```

Comment:
The title of this section seems very broad.  "End user application development" could mean a wealth of things from web applications, to games, to finance dashboards.  However the example given seems to be more about message queues for one-off tasks in a web server.  We should probably change the title.